main.tf

```diff
@@ -135,6 +161,19 @@ module "alb_https_sg" {
   tags = "${local.tags}"
 }
 
+module "alb_http_sg" {
+  source  = "terraform-aws-modules/security-group/aws//modules/http-80"
+  version = "v2.9.0"
+
+  name        = "${var.name}-alb-http"
+  vpc_id      = "${local.vpc_id}"
+  description = "Security group with HTTP ports open for everybody (IPv4 CIDR), egress ports are all world open"
```

Comment:
`open for everybody` in the description should be changed to reflect the reality.

Reply:
Well... it's as open as the HTTPS security group to allow to respond to any request with the redirect to HTTPS.

Reply:
Yes, I understand. I thought it should be more correct like this - "Security group with HTTP ports open for specific IPv4 CIDR block (or everybody), egress ports are all world open". Same for `https`.